app/assets/javascripts/_stick-at-top-when-scrolling.js

```diff
@@ -0,0 +1,19 @@
+(function(GOVUK, GDM) {
+
+  GOVUK.GDM = GDM;
```

Comment:
I don't know, actually.  It's basically like the fake experiment about [the monkeys and the ladder](http://www.wisdompills.com/2014/05/28/the-famous-social-experiment-5-monkeys-a-ladder/) now.